client/my-sites/email/email-providers-comparison/price/google-workspace.tsx

```diff
@@ -27,6 +32,55 @@ const getGoogleWorkspaceProductSlug = ( intervalLength: IntervalLength ): string
 		: GOOGLE_WORKSPACE_BUSINESS_STARTER_YEARLY;
 };
 
+const AdditionalPriceInformation = ( {
+	currencyCode,
+	product,
+}: {
+	currencyCode: string | null;
+	product: ProductListItem | null;
+} ): ReactElement | null => {
+	if ( ! hasDiscount( product ) ) {
+		return null;
+	}
+
+	const standardPriceForIntervalLength = formatPrice( product?.cost ?? 0, currencyCode ?? '' );
+	const salePriceForIntervalLength = formatPrice( product?.sale_cost ?? 0, currencyCode ?? '' );
```

Comment:
Those two variable names are a bit weird to me. We don't support any other interval rather than yearly. Should those variable just be called...

```suggestion
	const standardPrice = formatPrice( product?.cost ?? 0, currencyCode ?? '' );
	const salePrice = formatPrice( product?.sale_cost ?? 0, currencyCode ?? '' );
```

Even if we support it in the future, it looks to me that we don't need to suffix it.

Reply:
That's a very good point 👍.